src/Notifiers/Slack/SlackNotifier.php

```diff
@@ -168,24 +225,19 @@ private function buildConferencesBlocks(array $conferences): array
         ];
     }
 
-    /** @return array<array> */
-    private function buildCfpsBlocks(): array
+    /**
+     * @param array<mixed> $endingCfps
+     *
+     * @return array<array>
+     * */
+    private function buildCfpsBlocks(array $endingCfps): array
     {
         $cfpsBlock = [
             $this->slackBlocksBuilder->buildHeader('Ending soon CFPs'),
             $this->slackBlocksBuilder->buildDivider(),
         ];
 
-        $endingCfps = $this->getEndingCfps();
-
-        if (
-            0 === \count($endingCfps[0]) &&
-            0 === \count($endingCfps[1]) &&
-            0 === \count($endingCfps[5]) &&
-            0 === \count($endingCfps[10]) &&
-            0 === \count($endingCfps[20]) &&
-            0 === \count($endingCfps[30])
-        ) {
+        if (0 === \count($endingCfps)) {
             $cfpsBlock[] = $this->slackBlocksBuilder->buildSimpleSection('No ending CFP to display today !');
```

Comment:
If there is nothing to display, we display nothing (not a test to say we have nothing to display).
Same in buildCfpsBlocks